packages/cadl-ranch-specs/http/azure/core/facet/mockapi.ts

```diff
@@ -0,0 +1,23 @@
+import { passOnSuccess, mockapi, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+const priceResult = { value: 15, count: 1 };
+const expectInputBody = {
+  kind: "Int32Values",
+  ranges: [10, 20],
+  value: 15,
+  field: "price",
+};
+
+const responseBody = {
+  field_name: "price",
+  results: [priceResult],
+};
+
+Scenarios.Azure_Core_Facet_getInt32Facets = passOnSuccess(
```

Comment:
it's unclear what testing a facet is about testing. What are you trying to test in `azure.core`?

Reply:
Regarding the facet related operations in Models-TypeSpec, I had some misunderstandings. There are actually no new features. I will close this PR